src/main/java/uk/gov/companieshouse/confirmationstatementapi/configuration/InterceptorConfig.java

```diff
@@ -21,6 +22,7 @@
 
     static final String TRANSACTIONS = "/transactions/**";
     static final String FILINGS = "/private/**/filings";
+    static final String SUBMISSIONS = TRANSACTIONS + "/confirmation-statement/{confirmation_statement_id}/**";
```

Comment:
you can blame me for that,khese was gitting null pointer we think because the ConfirmationStatementController.createNewSubmission method was matching 
TRANSACTIONS + "/confirmation-statement/**
I suggested adding the {confirmation_statement_id} to try and khese said it seemed to work.

Reply:
If it is working, leave it as is then